### PR TITLE
Added mrr column to subscriptions

### PR DESCRIPTION
**Pull request by SimonBackx**
refs https://github.com/TryGhost/Team/issues/1457

We want to save the MRR with a subscription to simplify the calculation of the total MRR once, in 5.0, we also take 'forever' offers into account into the MRR (so we can just SUM the MRR of all subscriptions).

- Sets the MRR to 0 for now.
- A migration (in a different PR) will fill in all the values in a data migration, but this needs to get merged first because we need this new column in order to update the members-api package (so we already save the MRR before doing the data migration).

### CODE REVIEW

File: core/server/data/migrations/versions/4.43/2022-04-08-14-41-add-mrr-to-subscriptions.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('members_stripe_customers_subscriptions', 'mrr', {
```
Comment (matthanley) [inline core/server/data/migrations/versions/4.43/2022-04-08-14-41-add-mrr-to-subscriptions.js:3]: Are we sure we want to go with MRR vs ARR?

Comment (SimonBackx) [reply core/server/data/migrations/versions/4.43/2022-04-08-14-41-add-mrr-to-subscriptions.js:3]: Discussed in https://ghost.slack.com/archives/C02G9E68C/p1649667605432439?thread_ts=1649408400.454149&cid=C02G9E68C

tl;dr: we'll use MRR.

**Comment by SimonBackx** [comment]
Should I bump the migration version to v4.44 or is that not required?

**Comment by daniellockyer** [comment]
@SimonBackx Yep, you'll need to bump it 🙂

**Comment by daniellockyer** [comment]
@SimonBackx Do you have the associated `schema.js` change for this as well? 🙂

**Comment by SimonBackx** [comment]
@daniellockyer I updated the schema.js file, and also fixed the tests 😊

**Review by daniellockyer — APPROVED** [review]

**Review by matthanley — APPROVED** [review]

**Review by allouis — APPROVED** [review]
